docs/editors/vim.md

```diff
@@ -34,13 +22,12 @@ sure you have the correct version installed.
 ```sh
 # If using Vim
 vim --version | head
-VIM - Vi IMproved 8.1
```

Comment:
Should we recommend nvim over vim?

Reply:
8.2 is working pretty well right now. All of the same core functionality needed for a pleasant LSP experience is there, albiet I've found some of it to be not as smooth. Either way, coc.nvim promises to work with both, so I'd say for now, we can leave both in.